test/e2e_env/universal/mtls/mtls.go

```diff
@@ -64,23 +64,23 @@ mtls:
 			Expect(err).ToNot(HaveOccurred())
 		}
 		It("PERMISSIVE server without TLS", func() {
-			setupTest("PERMISSIVE", false)
+			setupTest("PERMISSIVE")
 			By("Check inside-mesh communication")
 			trafficAllowed("test-server.mesh")
 
 			By("Check outside-mesh communication")
 			trafficAllowed(publicAddress())
 		})
 		It("STRICT server without TLS", func() {
-			setupTest("STRICT", false)
+			setupTest("STRICT")
 			By("Check inside-mesh communication")
 			trafficAllowed("test-server.mesh")
 
 			By("Check outside-mesh communication")
 			trafficBlocked(publicAddress())
 		})
 		It("PERMISSIVE server with TLS", func() {
-			setupTest("PERMISSIVE", true)
+			setupTest("PERMISSIVE")
```

Comment:
the flag was never used so isn't this test the same as `PERMISSIVE server without TLS` ? or is it set up somewhere else?

Reply:
Yeah, it was broken in https://github.com/kumahq/kuma/pull/4667. fixed it

Reply:
@slonka it's apparently not running successfully, I'm going to create an issue to fix it and keep the current behavior.

Reply:
can you link to the issue when it's done?

Reply:
https://github.com/kumahq/kuma/issues/4864